src/lib/form/useOnSubmit.ts

```diff
@@ -58,8 +58,10 @@ export const useOnSubmitEdit = <TFormValues extends IdentifiableObject>({
 
 export const useOnSubmitNew = <TFormValues>({
     section,
+    valueFormatter,
 }: {
     section: ModelSection
+    valueFormatter?: (values: TFormValues) => Record<string, unknown>
```

Comment:
Thank you! I was planning to do something similiar, where we could use the output of `zod.parse`, and get the benefit of `.trim()` etc.